indigo_app/static/javascript/indigo/models.js

```diff
@@ -135,7 +135,8 @@
       if (Indigo.Preloads.provisionEid && Indigo.Preloads.provisionEid.lastIndexOf('__') > -1) {
         eidPrefix = Indigo.Preloads.provisionEid.substring(0, Indigo.Preloads.provisionEid.lastIndexOf('__'));
       }
-      new indigoAkn.EidRewriter().rewriteAllEids(this.xmlDocument.documentElement, eidPrefix);
+      new indigoAkn.EidRewriter(Indigo.Preloads.provisionCounters, Indigo.Preloads.eidCounter)
```

Comment:
You can forego all changes to EidRewriter by doing something like:

```js
rewriter = new indigoAkn.EidRewriter();
rewriter.counters = Indigo.Preloads...
rewriter.eidCounter = Indigo.Preloads...
rewrite.rewriteEid(this.xmlDocument.documentElement, eidPrefix);
```

Reply:
woah! I can!